config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -213,7 +213,7 @@ periodics:
   spec:
     containers:
     - args:
-      - --timeout=1080
+      - --timeout=1200
```

Comment:
Is 1200 enough?

Reply:
Increased to 22h. You are right, it's better to be safe than sorry.